modules/data.atmosphere/R/metgapfill.R

```diff
@@ -383,7 +383,7 @@ metgapfill <- function(in.path, in.prefix, outfolder, start_date, end_date, lst
     nelem <- length(time)
     tunit <- ncdf4::ncatt_get(nc = nc, varid = "time", attname = "units", verbose = verbose)
     origin <- "1900-01-01 00:00:00"
-    time <- round(as.POSIXlt(udunits2::ud.convert(time, tunit$value, paste("seconds since", origin)),
+    time <- round(as.POSIXlt(PEcAn.utils::ud_convert(time, tunit$value, paste("seconds since", origin)),
                              origin = origin, tz = "UTC"), units = "mins")
     dtime <- diff(time)
```

Comment:
I think this should fix the "possible error" warning:
```suggestion
                             origin = origin, tz = "UTC"))
    dtime <- as.numeric(diff(time), units = "mins")
```